__tests__/ncgamesapp.test.js

```diff
@@ -0,0 +1,24 @@
+const app = require("../ncgamesapp.js");
+const request = require("supertest");
+const connection = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return connection.end();
+});
+describe("GET:/api/categories", () => {
+  it("GET api/categories responds with categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.categories).toHaveLength(4);
```

Comment:
Your tests here are only checking length of the array

You should test the contents are correct too!